internal/circonus/circonus.go

```diff
@@ -364,6 +364,11 @@ func NewMetricDestination(opts *MetricDestConfig, logger cua.Logger) (*trapmetri
 		checkTags = append(checkTags, getOSCheckTags()...)
 	}
 
+	if ch.circCfg.CheckTags != "" {
+		confCheckTags := strings.Split(ch.circCfg.CheckTags, ":")
```

Comment:
If you are going to have check tags in the config be a comma delimited string instead of an array (which I don't recommend, but maybe you have to do it that way), wouldn't you need to split on "," not ":" to get the individual tags? Each tag is a `category:value` pair that needs to stay together. This would split them on the colons and append weird strings containing `valuePrevTag,categoryNextTag`. I would still recommend having the config value just be a []string slice and then just `checkTags = append(checkTags, ch.circCfg.CheckTags...)` though.

Reply:
Good looks, I must have mixed up "," for ":". Thanks!

This is the way the kubernetes agent does it. I'm sure it would be trivial to change both to ensure consistency.